packages/x-date-pickers/src/PickersTextField/PickersInputBase/PickersInputBase.tsx

```diff
@@ -21,14 +21,16 @@ import {
   Unstable_PickersSectionListSectionSeparator as PickersSectionListSectionSeparator,
   Unstable_PickersSectionListSectionContent as PickersSectionListSectionContent,
 } from '../../PickersSectionList';
+import { usePickerTextFieldOwnerState } from '../usePickerTextFieldOwnerState';
+import { PickersTextFieldOwnerState } from '../PickersTextField.types';
 
 const round = (value: number) => Math.round(value * 1e5) / 1e5;
 
 export const PickersInputBaseRoot = styled('div', {
   name: 'MuiPickersInputBase',
   slot: 'Root',
   overridesResolver: (props, styles) => styles.root,
-})<{ ownerState: OwnerStateType }>(({ theme }) => ({
+})<{ ownerState: PickersTextFieldOwnerState }>(({ theme }) => ({
```

Comment:
Just noticed this:
Do we intentionally use "TextField" here to show the meaning of the overall concept and "input" elsewhere to target the actual element?

Reply:
The Text Field is the component that contains the label, the error message etc... AND the input.

It's not a nomenclature I love tbh, but I tried to stay close from the core logic here to smoothen the migration :+1: